Sources/Tracking/CustomerIO.swift

```diff
@@ -98,6 +100,32 @@ public extension CustomerIOInstance {
                  jsonEncoder: jsonEncoder)
     }
 
+    func identify(
```

Comment:
IMO, having 2 functions called `identify` is confusing to the customer wondering when to use what function. I propose having `identify()` stay as it is in the SDK to identify a customer profile and something like:

```swift
customerio.setProfileAttributes(["first_name": "Dana"], onComplete: ...)

// with the background queue, we can do synchronous methods like:
customerio.profileAttributes = ["first_name": "Dana"]
```

Some function naming that implies that 1 function's job is to identify a profile and 1 function's job is to alter attributes of an already identified profile.

Reply:
👍 had to get past my preconceive notions of `identify` on this one, since it's always been the endpoint name, but I agree, and since it's `identify + logic` makes sense to treat it slightly different